content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,374 @@
+=============
+Create events
+=============
+
+With Odoo *Events*, users can create and configure custom events of any size, for any occasion.
+
+New event
+=========
+
+There are a number of ways to create events directly from the :guilabel:`Events` dashboard, with
+each view option having its own process. To learn more about that check out the :doc:`Event
```

Comment:
Yes, it was my intention to say 'view option' in this case -- as in, the different view options users can utilize to see the dashboard in different ways. I'll add some specifics to this sentence in order to avoid any potential confusion. Thanks for pointing this out!